lms/views/api/exceptions.py

```diff
@@ -254,10 +254,13 @@ def __json__(self, request):
                 # If we don't have an access token we can't refresh it.
                 pass
             else:
-                body["refresh"] = {
-                    "method": "POST",
-                    "path": request.route_path("canvas_api.oauth.refresh"),
-                }
+                if request.matched_route.name.startswith("canvas_api."):
```

Comment:
We could eventually use prefixes to organize the routes and maybe it's possible to check that (instead of name.startswith) 

https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#using-a-route-prefix-to-compose-applications


I don't think `name.startswith` is problematic here at the moment in any case.